falcon/statics.py

```diff
@@ -0,0 +1,43 @@
+import copy
+import mimetypes
+import os.path
+
+import falcon
+
+
+class StaticSink:
+
+    _additional_types = {
+        '.css': 'text/css',
+        '.js': 'application/javascript',
+        '.html': 'text/html',
+        '.svg': 'image/svg+xml',
+
+        '.png': 'image/png',
+        '.jpeg': 'image/jpeg',
+        '.jp2': 'image/jpeg',
+        '.gif': 'image/gif',
+        '.tiff': 'image/tiff',
+    }
+
+    def __init__(self, prefix, parent_folder):
+        self.prefix = prefix
+        self.parent_folder = os.path.abspath(parent_folder)
+        self.mime_types = copy.copy(mimetypes.encodings_map)
```

Comment:
Also, I think it would be helpful to factor this out to initialize a new option, `falcon.ResponseOptions.static_media_types`, and use that here. Then, people can add additional types to that dict as needed.